scripts/hic2sci.sh

```diff
@@ -8,7 +8,7 @@ for ((i=1;i<=22;i++)); do
 	for ((j=$((i))+1;j<=22;j++)); do 
 		c1=chr${i}
 		c2=chr${j}
-		java -jar ${JUICERTOOLS} dump observed GW_KR ${infile} chr${i} chr${j}  BP ${res} | awk -v c1=chr${i} -v c2=chr${j} -v res=${res} '{OFS="\t"}{print c1,$1,$1+res,c2,$2,$2+res,$3}' >> ${outfile};
+		java -jar ${JUICERTOOLS} dump observed KR ${infile} chr${i} chr${j}  BP ${res} | awk -v c1=chr${i} -v c2=chr${j} -v res=${res} '{OFS="\t"}{print c1,$1,$1+res,c2,$2,$2+res,$3}' >> ${outfile};
```

Comment:
Fix4.

Reply:
Hi, 
SCI originally works with GenomeWide-KR normalized (GW_KR). Check Juicer tools addNorm command to add the normalization.

Thanks
Haitham

Reply:
ok, thanks.

Reply:
I can merge other Fixes if you can reverse this change. 

Thanks
Haitham